### ISSUE TITLE
Meaning lists need re-ordering in case of changes vis à vis original list

### ISSUE THREAD

**Issue by PaulHeggarty** [lingdb/CoBL#136]
In the reference list for Jena 200, chest still appears between bone and breathe, because it used to be breast.  We need to think of how to handle this.

**Comment by cormacanderson** [comment]
@runjak I don't think we have a way of changing this on the website as things stand. If I send you the same excel sheet as before with the correct sort order of the relevant lists, along with ID numbers, is it easy enough for you to change them?

**Comment by PaulHeggarty** [comment]
Let's not get Jakob to do this now, it's not a priority.  This is not necessarily the right way anyway, I think.  This issue is for us two to sort out first, then get Jakob onto it.

**Comment by runjak** [comment]
Changing the sort order on my site when given an excel sheet should be fairly simple, I think.
